README.md

```diff
@@ -77,6 +77,27 @@ bin/dev
 
 This will start the development server and background worker. Open your web browser and go to http://localhost:3000 to see the application running.
 
+##Working with Devise Invitable
+If you need to work with the user registration/invitation process, follow these steps:
+
+1. Make sure you are logged out of the application.
+
+2. Open a Rails console session:
+```bash
+rails console
+```
+3. In the console, run the following command to invite a user:
+```bash 
+User.invite!(email: <some_test_email>)
+```
+This will send an invitation email with an invite link.
```

Comment:
Can you include here that for local development we use `letter_opener` and how to access that.

Reply:
I have made changes according to your instructuions  you can review it.